docs/automated-process.md

```diff
@@ -0,0 +1,30 @@
+# Automated Ansible Release Process
+
+## Preamble
+
+This document describes the (mostly) automated ansible community package release process.
+
+!!! note
+    Throughout this page, placeholder values in code blocks are formatted as
+    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to fill in.
+
+
+## Trigger the workflow
+
+1. Trigger [the automated workflow on GHA](https://github.com/ansible-community/ansible-build-data/actions/workflows/ansible-release.yml) and fill in the release version (like 11.2.0) and the major version (like 11).
+   The process will create a PR in the `ansible-build-data` repository, and wait for approval before continuing with uploading the package to PyPI.
+
+2. Check out the PR in your `ansible-build-data` clone and copy the updated porting guide into the `ansible-documentation` repository.
```

Comment:
Depending on the audience, you might want to fill in details - for example, the path to the updated porting guide, the path where it should go in the ansible-documentation repo.

Reply:
Looks great, thanks @felixfontein.